conversions/hex-bin.py

```
@@ -0,0 +1,57 @@
+#  CONVERT HEXADECIMAL TO BINARY
+import math
+
+
+def hex_to_bin(hex_num: str) -> int:
+    """
+    Convert a hexadecimal value to its decimal equivalent
+    #https://stackoverflow.com/questions/1425493/convert-hex-to-binary
+    Here, we have used " >> " bitwise operator
+    Bitwise right shift: 
+    Shifts the bits of the number to the right and fills 0 on voids left as a result. 
+    Similar effect as of dividing the number with some power of two.
+    Example: 
+    a = 10
+    a >> 1 = 5 
+    
+    >>> hex_to_bin("AC")
+    10101100
+    >>> hex_to_bin("9A4")
+    100110100100
+    >>> hex_to_bin("   12f   ")
+    100101111
+    >>> hex_to_bin("FfFf")
+    1111111111111111
+    >>> hex_to_bin("F-f")
+    False
+    >>> hex_to_bin("")
+    False
+    """
+
+    bin_str: str = ""
+    hex_str: str = hex_num.strip()
+
+    if not hex_str:
+        return False
```

Same as below. Simply raise a `ValueError` instead of returning a boolean